docs/guides/multiverse-computing-singularity.ipynb

```diff
@@ -1059,6 +1065,31 @@
    "id": "cddff465",
    "metadata": {},
    "source": [
+    "## Changelog\n",
+    "\n",
+    "#### 4th June 2025\n",
```

Comment:
With HTML, we need to avoid skipping heading levels for accessibility reasons. It should go h2 -> h3 -> h4, not h2 -> h4.

```suggestion
    "### 4th June 2025\n",
```